src/spatio_temporal_voxel_layer.cpp

```diff
@@ -334,6 +349,35 @@ void SpatioTemporalVoxelLayer::PointCloud2Callback( \
   buffer->Unlock();
 }
 
+/*****************************************************************************/
+void SpatioTemporalVoxelLayer::BufferEnablerCallback(   \
+                std_srvs::SetBool::Request & request,   \
+                std_srvs::SetBool::Response & response, \
+                boost::shared_ptr<buffer::MeasurementBuffer>& buffer, \
+                boost::shared_ptr<message_filters::SubscriberBase>& subcriber)
+/*****************************************************************************/
+{
+  buffer->Lock();
+  if( buffer->isEnabled() != request.data ) //updated
+  {
+    buffer->setEnabled(request.data);
+    if(request.data == true)
```

Comment:
`if(request.data)`, the true is redundant

Reply:
The service can be true or false. To enable /disable the observation.  With that if I check  if you want to activate or deactivate the observation.

Reply:
Sorry, what i mean is the line can be `if(request.data)` instead of `if(request.data == true)`